src/server/api/cla.js

```diff
@@ -169,15 +169,20 @@ function updateUsersPullRequests(args) {
         if (err || !user || !user.requests || user.requests.length < 1) {
             let req = {
                 args: {
-                    repo: args.repo,
-                    owner: args.owner,
                     gist: args.item.gist,
-                    sharedGist: args.item.sharedGist,
                     token: args.token
                 }
             };
-            ClaApi.validatePullRequests(req, () => { });
```

Comment:
@KharitonOff  Do you specifically just want to check all the repo pull request instead of org pull requests or shared CLA orgs/repos?

Reply:
@MichaelTsengLZ actually yes, my intention was to reduce the number of calls, because this function is called every time a user signs a CLA. I assumed, that in 90% of cases the user has only one open PR in the certain repo, where he/she comes from to sign the CLA. In other cases I would take in count that the repo/org owner has to trigger "Recheck PRs" button manualy.
But since we use user-PR cache it should be ok, to check all PRs of the linked item as a fallback.